pkg/gator/test/test.go

```diff
@@ -173,16 +172,6 @@ func Test(objs []*unstructured.Unstructured, tOpts Opts) (*GatorResponses, error
 	return responses, nil
 }
 
-func isTemplate(u *unstructured.Unstructured) bool {
-	gvk := u.GroupVersionKind()
-	return gvk.Group == templatesv1.SchemeGroupVersion.Group && gvk.Kind == "ConstraintTemplate"
-}
-
-func isConstraint(u *unstructured.Unstructured) bool {
-	gvk := u.GroupVersionKind()
-	return gvk.Group == "constraints.gatekeeper.sh"
-}
-
```

Comment:
Do we not have helper methods like these in the code somewhere already? Perhaps not but they seem to straightforward and helpful i'd be surprised.

Reply:
That's a great question lol, do you remember seeing anything back when these were written in [the gator test](https://github.com/open-policy-agent/gatekeeper/pull/1786/files#diff-8bd2f5d4ed03ca9981ae6a3b797f90d8c2d89c0e619016c3c699ecf0b499e395R73) PR